### ISSUE TITLE
Perhaps just port the engine

### ISSUE THREAD

**Issue by thekiltedartist** [daniele-rapagnani/alephone-android#25]
First of all, i want to say great job on doing what the main Aleph One community won't, and that is porting it to android. I realize this is still in the early stages, but perhaps instead of porting each individual game, just port the aleph one engine. I'm thinking something similar to Delta/Quad Touch on android for Doom and Quake, where you can select the game you want to play. This also opens it up to other mods and TCs.
Not that i have played with this port much, but if it hasn't been included, contoller support would be great.

Again, great job and keep up the good work!

**Comment by daniele-rapagnani** [comment]
Hey there, thanks for reaching out.

> but perhaps instead of porting each individual game, just port the aleph one engine
That's what I did! I just build individual applications for each game just for convenience, so that users can play them without having to download asset files and place them in the right location on internal/external storage. The source code is the same for each game though.

> This also opens it up to other mods and TCs.
If that's not already possibile it should be an easy fix, I'll add an issue on this specifically.

> Not that i have played with this port much, but if it hasn't been included, contoller support would be great.
Yes, controller support is included. Most people seem to use a Dualshock 4 but I also tried with other bluetooth controllers with no issue so far.
